base/applications/mspaint/dialogs.cpp

```diff
@@ -162,18 +164,19 @@ LRESULT CAttributesDialog::OnRadioButton3(WORD wNotifyCode, WORD wID, HWND hWndC
 
 LRESULT CAttributesDialog::OnEdit1(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL& bHandled)
 {
+    FixDpi();
```

Comment:
Can't you just call this function at the only place where those `g_xDpi` and `g_yDpi` are (re)initialized at runtime?

Reply:
`FixDpi()` is deleted.